### ISSUE TITLE
Unable to launch 2.2. miner on Ubuntu 24.04 LTS

### ISSUE THREAD

**Issue by mchilson** [vertcoin-project/one-click-miner-vnext#636]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read [Common issues and fixes](https://github.com/vertcoin-project/one-click-miner-vnext/issues/618)?.

- [X] Yes / Not relevant

### Current behaviour

When launching the GUI version from the terminal I get this error and the program will not launch. 

vertcoin-ocm: error while loading shared libraries: libwebkit2gtk-4.0.so.37: cannot open shared object file: No such file or directory

### Expected behaviour

The program should run.

### Steps to reproduce

Try and launch the miner on Ubuntu 24.04 LTS from the terminal or desktop icon—same result.

### How did you obtain Vertcoin OCM?

Pre-built binaries

### What version of Vertcoin OCM are you using?

v2.2

### Operating system and version

Ubuntu Budgie 24.04.1 LTS

### Machine specifications

I7 - 32 gig of RAM
NVIDIA video

**Comment by KforG** [comment]
Try installing the libwebkit2gtk version the error lists.

**Comment by mchilson** [comment]
Yes, I tried that. That version of the library is no longer available.

**Comment by KforG** [comment]
I see it's not possible on 24.04.
I think this will be solved by upgrading wails to the 3.0 version, but this seem like a rather extensive task. Which I can't commit the time to right now.

For the time being if you want to mine VTC I would recommend using Verthashminer directly.

**Comment by Nisse43** [comment]
I made a quick port to Wails 2 as a first step which now works on my machine. Unfortunately there is alot of extra changes also but mostly npm noice I think.

https://github.com/vertcoin-project/one-click-miner-vnext/pull/650
